homeassistant/const.py

```diff
@@ -434,6 +435,9 @@
 ELECTRIC_POTENTIAL_MILLIVOLT: Final = "mV"
 ELECTRIC_POTENTIAL_VOLT: Final = "V"
 
+# Power factor units
+ELECTRIC_POWER_FACTOR: Final = "pf"
+
```

Comment:
I don't think pf / power factor is a valid unit.

Reply:
Power factor is the ratio of working power to apparent power so actually it has no unit of measurement.
It can be uppercase if you like it but I believe it makes no difference

Reply:
Remove the invalid unit
```suggestion
```